tests/service_tests/test_domain.py

```diff
@@ -52,3 +52,23 @@ def testGetInfo(self, mock_client):
         ds.update_cookie.assert_called_with({"cookie":"value"})
         i.service.getInfo.assert_called_with('example.com')
         self.assertEqual(result, [ 'foo' ])
+
+    @patch('suds.client.Client')
+    def testSetDnsEntries(self, mock_client):
+        # SETUP
+        ds = DomainService()
+        ds.build_cookie = Mock(return_value={"cookie": "value"})
+        ds.update_cookie = Mock()
+
+        i = mock_client.return_value
+        i.service.setDnsEntries.return_value = None
+
+        dns_entry = DnsEntry('testentry', 86400, DnsEntry.TYPE_A, '127.0.0.1')
+
+        # CALL
+        result = ds.set_dns_entries('domain1', [dns_entry, ])
```

Comment:
In both cases the server replies with 'Invalid API signature, signature does not match the request.', which is a suds.WebFault and can be catched.